middle_end/flambda2/simplify/env/closure_info.ml

```diff
@@ -19,31 +19,44 @@ type t =
   | Closure of
       { code_id : Code_id.t;
         return_continuation : Continuation.t;
-        exn_continuation : Continuation.t
+        exn_continuation : Continuation.t;
+        self_continuation : Continuation.t;
```

Comment:
I would maybe call it `self_tail_call_continuation`

Reply:
Now called `tailrec_to_cont` and part of a new datatype, stored in `Downwards_env` instead.